std-ddox.ddoc

```diff
@@ -3,6 +3,9 @@ XREF = std.$1.$2
 XREF_PACK = std.$1.$2.$3
 CXREF = core.$1.$2
 ECXREF = etc.$1.$2
+MREF_HELPER=/$1$(MREF_HELPER $+)
```

Comment:
Could you please rename this as `SLASH_PREFIXED`? I reckon there are other good uses for it.